Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated_NetCoreVS_v4.x.nuspec

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.Azure.Functions.Worker.ItemTemplates.NetCoreVS</id>
```

Comment:
Thoughts on name + versioning for new package? Would also like reviews for the other newly introduced package.